manifests/broker.pp

```diff
@@ -0,0 +1,20 @@
+# Set up the broker
+class pulp::broker {
+  if $pulp::manage_broker {
```

Comment:
I am open to arguments as to why it should not; my personal take on it has always been that you should be able to configure things like brokers yourself, however, there should be an 'easy' button where Pulp configures everything it needs with sane defaults. In our current usage of the 0.1-stable branch, we rely on being able to do things like -- https://github.com/Katello/katello-installer/blob/master/modules/pulp/manifests/init.pp#L130 to customize to our needs.